### PR TITLE
259492 confirm waste record complete

### PR DESCRIPTION
**Pull request by AtosSaqlain**

### CODE REVIEW

File: Portal/Controllers/WasteController.cs [hunk 1]
```diff
@@ -93,5 +93,16 @@ public async Task<IActionResult> Types(WasteTypesViewModel wasteTypesViewModel)
 
             return RedirectToAction("Index", "Home");
         }
+
+        [HttpGet]
+        [Route("/WasteRecordStatus")]
```
Comment (NikPeters-Atos) [inline Portal/Controllers/WasteController.cs]: Route should proably be RecordStatus or even just Status as it's already on the WasteController

File: Portal/Services/WasteService.cs [hunk 2]
```diff
@@ -136,5 +137,18 @@ public async Task SaveSelectedWasteType(int journeyId, string selectedWasteType)
 
             await _httpWasteService.SaveSelectedWasteType(journeyId, selectedWasteType);
         }
+
+        public async Task<WasteRecordStatusViewModel> GetWasteRecordStatus(int journeyId)
+        {
+            var result = await _httpWasteService.GetWasteRecordStatus(journeyId);
+            return new WasteRecordStatusViewModel
```
Comment (NikPeters-Atos) [inline Portal/Services/WasteService.cs]: Probably worth using auto mapper here isn't it?

File: Portal/Views/Waste/WasteRecordCompleteStatus.cshtml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+<div class="text-center">
+    <span class="govuk-body govuk-!-font-weight-bold">@WasteRecordCompleteResources.WhatToDoNext</span><br /><br />
+    <span class="govuk-caption-m">@WasteRecordCompleteResources.AccountIsActive <a asp-action="Index">@WasteRecordCompleteResources.AccountLink</a></span><br />
+    <span class="govuk-caption-m">@WasteRecordCompleteResources.WasteBalance @Model.WasteBalance @WasteRecordCompleteResources.WasteBalance2</span><br />
+</div>
+<br />
+
+<button class="govuk-button" data-module="govuk-button">
+    @WasteRecordCompleteResources.ContinueButton
```
Comment (NikPeters-Atos) [inline Portal/Views/Waste/WasteRecordCompleteStatus.cshtml]: Continue button text has been provided in MasterResources

File: Portal/Services/WasteService.cs (hunk 2 above)
Comment (AtosSaqlain) [reply Portal/Services/WasteService.cs]: Yeah, I wanted to get this out the door in the current sprint, so will add automapper in the next story.

File: Portal/Controllers/WasteController.cs (hunk 1 above)
Comment (AtosSaqlain) [reply Portal/Controllers/WasteController.cs]: Agreed. Actually, shouldn't we rename the controller to "Journey" or "WasteJourney"? That would be more descriptive and intuitive. Thoughts?

File: Portal/Views/Waste/WasteRecordCompleteStatus.cshtml (hunk 3 above)
Comment (AtosSaqlain) [reply Portal/Views/Waste/WasteRecordCompleteStatus.cshtml]: Done

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**